.github/workflows/release-artifacts.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - uses: actions/checkout@master
+    - uses: actions/checkout@main
```

Comment:
i think this should still be master since it refers to the version of `actions/checkout`